frontend/packages/noobaa-storage-plugin/src/table-filters.ts

```diff
@@ -0,0 +1,52 @@
+import * as _ from 'lodash';
+import { Filter } from '@console/shared';
+import { getOBPhase, getOBCPhase } from './utils';
+
+const allPhases = ['Pending', 'Bound', 'Lost'];
+const PHASE_LOST = 'Lost';
+
+export const obcStatusFilter: Filter = {
+  type: 'obc-status',
+  selected: allPhases,
+  reducer: getOBCPhase,
+  items: _.map(allPhases, (phase) => ({
+    id: phase,
+    title: phase,
+  })),
+  filter: (phases, obc) => {
+    if (!phases || !phases.selected) {
+      return true;
+    }
+    let phase;
+    if (!obc.status) {
+      phase = PHASE_LOST;
+    } else {
+      phase = _.get(obc, 'status.Phase');
+      phase = phase.charAt(0).toUpperCase() + phase.substring(1);
+    }
+    return phases.selected.has(phase) || !_.includes(phases.all, phase);
+  },
+};
+
+export const obStatusFilter: Filter = {
+  type: 'obc-status',
```

Comment:
should be like 
```suggestion
  type: 'ob-status',
```

right ?

Reply:
ack